ar_commands.rb

```diff
@@ -5,27 +5,29 @@
 require_relative "models/painting"
 
 # 1. Create a new Artist (Vincent Van Gogh) and save him to your database
-
+Artist.new(name: "Vincent van Gogh")
 # 2. Create a new Painting (Starry Night) and save it to your database
-
+Paintings.new(title: "Starry Night")
 # 3. Return all Artists
-
+Artist.all
 # 4. Return all Paintings
-
+Painting.all
 # 5. Return the artist with an id of 2
-
+Artist.find_by(id: 2)
 # 6. Return the artist with a name of "Vincent Van Gogh"
-
+Artist.find_by(name: "Vincent Van Gogh")
 # 7. Return all the artists from Spain
-
+Artist.find_by(nationality: "Spanish")
 # 8. Return the painting with an id of 1
-
+Painting.find_by(id: 1)
 # 9. Return the painting with a name of "Guernica"
-
+Painting.find_by(title: "Guernica")
 # 10. Return all the paintings that "Pablo Picasso" painted
-
+Painting.find_by(artist_id: cube.id)
 # 11. Return the artist of "Guernica"
-
+guernica = Painting.find_by(title: "Guernica")
 # 12. Change Vincent Van Gogh's nationality to USA
-
+Artist.find_by(name: "Vincent Van Gogh")update(nationality: "USA")
```

Comment:
missing a `.` in your method chain...might be easier to break it into two lines:

```ruby
van_gogh = Artist.find_by(name: "Vincent Van Gogh")
van_gogh.update(nationality: "USA")
```